src/audio/Sound.js

```diff
@@ -926,6 +948,12 @@ FORGE.Sound.prototype.stop = function(internal)
         this._playing = false;
         this._paused = false;
 
+        if (this._loop !== true)
+        {
+            this._viewer.onPause.remove(this._onVisibilityChange, this);
+            this._viewer.onResume.remove(this._onVisibilityChange, this);
```

Comment:
I don't understand why the listener is binded to the pause and resume events of the viewer?